secboot/secboot_tpm.go

```diff
@@ -185,6 +185,41 @@ func MeasureSnapModelWhenPossible(findModel func() (*asserts.Model, error)) erro
 	return nil
 }
 
+// LockTPMSealedKeys manually locks access to the sealed keys. Meant to be
+// called in place of passing lockKeysOnFinish as true to
+// UnlockVolumeUsingSealedKeyIfEncrypted for cases where we don't know if a
+// given call is the last one to unlock a volume like in degraded recover mode.
+func LockTPMSealedKeys() error {
```

Comment:
(nitpick) my previous suggestion in https://github.com/snapcore/snapd/pull/9592#discussion_r517540281 to make this more linear still applies here. But it's correct so it's not a big deal :)

Reply:
I'll push a patch with that

Reply:
yes sorry I missed the change I made for this which was in another wip branch when I went to split out the secboot things yesterday, but thanks @bboozzoo for pushing it anyways :-)